### DISCUSSION TITLE
Setting Presets

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#5030] [Ideas]
I know this already exists, but it's only for a single setting. It would be useful for people who don't want to have to spend an hour going through each and every setting, for there just be a few presets that change every single setting. Preset ideas:

- Recommended — all the most recommended extensions for most users
- Advanced — all the most advanced and fancy extensions
- Bug Fixes — only with the extensions that fix bugs, and don't change the UI
- Beginner — for people who want a similar scratch experience to the original, but still some stuff different

If you wanted maybe you could create your own presets, but that's a separate idea.

**Comment by mxmou** [comment]
There should be a switch at the top of the "Recommended" section to enable all recommended addons, similar to what I suggested in #3496.
